oci/layout/oci_src.go

```diff
@@ -94,6 +95,7 @@ func (s *ociImageSource) GetManifest(ctx context.Context, instanceDigest *digest
 
 	m, err := ioutil.ReadFile(manifestPath)
 	if err != nil {
+		logrus.Errorf("Error HERE")
```

Comment:
@vrothberg Could you drop this part, please?

Reply:
Ouch, thanks for catching!

Reply:
https://github.com/containers/image/pull/1201